tests/BootstrapperTest.php

```diff
@@ -26,202 +22,187 @@
 use Stancl\Tenancy\Bootstrappers\DatabaseTenancyBootstrapper;
 use Stancl\Tenancy\Bootstrappers\FilesystemTenancyBootstrapper;
 
-class BootstrapperTest extends TestCase
-{
-    public $mockConsoleOutput = false;
```

Comment:
This might need to be added back in some way

Reply:
Hmm I'm thinking why the property was there. It is possible that tests are passing, but that the property needed to be set for some tests to work properly?

For example, some tests may be passing without actually making the necessary assertions now. So if the logic that they're testing broke, they'd still pass due to the property not being set.

Reply:
>  but that the property needed to be set for some tests to work properly?

I've checked the whole class. Firstly, `$mockConsoleOutput` is never being used (because we are not calling any command on those tests). 
Secondly, We just have one test [database_data_is_separated()](https://github.com/abrardev99/tenancy/blob/3.x/tests/BootstrapperTest.php#L49) where we are calling the `->artisan` command. And this test logic is pretty clear and looks good. I think everything is fine.

Reply:
`$mockConsoleOutput` was set in `BootstrapperTest` and `QueueTest`. Artisan is used in both.

> because we are not calling any command on those tests

https://github.com/archtechx/tenancy/blob/600bb823de185ef02c3ff5aeba0c75f41409bafa/tests/BootstrapperTest.php#L58

Reply:
I rechecked both classes. I don't see any usage or possible usage of the property.

Reply:
> I don't see any usage or possible usage of the property.

You're missing the point. It affects how artisan work. I showed you that artisan is called in those tests.

It's not about the property being used by us. It's about the property changing behavior of Laravel during the tests.

Reply:
Resolving this review in the favour #907.